### PR TITLE
[MXNET-344] [ONNX-MXNet] Add new Operator Translations for ONNX import module

### PR DESCRIPTION
**Pull request by anirudhacharya**
## Description ##
Add translations for the following operators.

- [x] Greater
- [x] Softplus
- [x] InstanceNormalization
- [x] Clip
- [x] Less
- [x] MaxRoiPool
- [x] Equal
- [x] LogSoftmax
- [x] Squeeze
- [x] ReduceLogSum
- [x] ReduceLogSumExp
- [x] ReduceSumSquare
- [x] Softsign
- [x] Not
- [x] Or
- [x] And
- [x] XOR

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- Add new operator translations

## Comments ##
- @spidyDev @Roshrini @sandeep-krishnamurthy

### CODE REVIEW

File: python/mxnet/contrib/onnx/_import/op_translations.py [hunk 1]
```diff
@@ -417,6 +458,16 @@ def flatten(attrs, inputs, proto_obj):
...
     return 'Flatten', new_attrs, inputs
 
+def clip(attrs, inputs, proto_obj):
+    """Clips (limits) the values in an array."""
+    new_attrs = translation_utils._fix_attribute_names(attrs, {'min' : 'a_min',
+                                                               'max' : 'a_max'})
+    if 'a_max' not in new_attrs:
+        new_attrs = translation_utils._add_extra_attributes(new_attrs, {'a_max' : np.inf})
+    if 'a_min' not in new_attrs:
+        new_attrs = translation_utils._add_extra_attributes(new_attrs, {'a_min' : -np.inf})
```
Comment (sandeep-krishnamurthy) [inline python/mxnet/contrib/onnx/_import/op_translations.py]: can we use np.ninf? I think that is float, please check once.

File: python/mxnet/contrib/onnx/_import/op_translations.py [hunk 2]
```diff
@@ -454,20 +505,49 @@ def reduce_mean(attrs, inputs, proto_obj):
...
 def reduce_prod(attrs, inputs, proto_obj):
-    """Reduce the array along a given axis by mean value"""
+    """Reduce the array along a given axis by product value"""
     new_attrs = translation_utils._fix_attribute_names(attrs, {'axes':'axis'})
     return 'prod', new_attrs, inputs
 
+def reduce_log_sum(attrs, inputs, proto_obj):
+    """Reduce the array along a given axis by log sum value"""
+    keep_dims = True if 'keepdims' not in attrs else attrs.get('keepdims')
+    sum_op = symbol.sum(inputs[0], axis=attrs.get('axes'),
```
Comment (sandeep-krishnamurthy) [inline python/mxnet/contrib/onnx/_import/op_translations.py]: Why inputs[0]?

File: python/mxnet/contrib/onnx/_import/op_translations.py (hunk 1 above)
Comment (anirudhacharya) [reply python/mxnet/contrib/onnx/_import/op_translations.py]: np.inf can be used as float - https://stackoverflow.com/questions/42315541/difference-between-np-inf-and-floatinf

File: python/mxnet/contrib/onnx/_import/op_translations.py (hunk 2 above)
Comment (anirudhacharya) [reply python/mxnet/contrib/onnx/_import/op_translations.py]: ReduceLogSum op takes has input tensor - https://github.com/onnx/onnx/blob/master/docs/Operators.md#ReduceLogSum. And while translating this ONNX operator into MXNet, we are splitting the reduceLogSum op to ``sum`` over a given axis and then ``log`` operator.

Comment (anirudhacharya) [reply python/mxnet/contrib/onnx/_import/op_translations.py]: it has been similarly used in other operator translations, for example here - https://github.com/apache/incubator-mxnet/pull/11140/files/6f518f13413387b40a5c45d96bd2c1d2decf7804#diff-ea46c490a749b158ef8986126da6bc12R284

**Pull request merged**